chromium_src/content/public/common/url_constants.h

```diff
@@ -6,7 +6,16 @@
 #ifndef BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_COMMON_URL_CONSTANTS_H_
 #define BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_COMMON_URL_CONSTANTS_H_
 
+#include "build/blink_buildflags.h"
+
+#if !BUILDFLAG(USE_BLINK)
+// Cannot include content deps
+namespace content {
+inline constexpr char kChromeUIScheme[] = "chrome";
+}
+#else
 #include "src/content/public/common/url_constants.h"  // IWYU pragma: export
+#endif
```

Comment:
Is it just some quick workaround?
Not having blink in content/public looks weird

Reply:
you're reading that backwards, `USE_BLINK` includes url_constants.h

Reply:
But that means ios build depends on `*content*` (at least by depending on `chromium_src/content/public/common/url_constants.h`)